.github/workflows/canary-integration-test.yml

```diff
@@ -1323,18 +1323,15 @@ jobs:
         run: |
           tests/scripts/github-action-helper.sh deploy_first_rook_cluster
           cd deploy/examples/
-          sed -i "/resources:/,/ # priorityClassName:/d" rbdmirror.yaml
+
           sed -i "/resources:/,/ # priorityClassName:/d" filesystem-mirror.yaml
-          kubectl create -f rbdmirror.yaml -f filesystem-mirror.yaml
```

Comment:
Why is `rbdmirror.yaml` commented out?  @subhamkrai

Reply:
it was removed by mistake, added now